test/IRGen/rdar146123129.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %target-swift-frontend -emit-ir -O -enable-library-evolution %s
+// REQUIRES: objc_interop
```

Comment:
Does this test require ObJC interop or is it that it requires the ObjC Foundation implementation and should be `// requires: OS=...`?